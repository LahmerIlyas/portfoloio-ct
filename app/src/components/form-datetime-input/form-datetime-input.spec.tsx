import React from 'react';
import renderer from 'react-test-renderer';
import { Formik } from 'formik';
import { FormDateTimeInput } from '.';

describe('<FormDateTimeInput />', () => {
  it('renders correctly', async () => {
    const tree = renderer
      .create(
        <Formik initialValues={{}} onSubmit={() => {}}>
          <FormDateTimeInput field="date" title="Date" />
        </Formik>,
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
