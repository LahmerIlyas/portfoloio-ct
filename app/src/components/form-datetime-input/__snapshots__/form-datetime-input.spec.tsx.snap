// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FormDateTimeInput /> renders correctly 1`] = `
<View
  style={
    Object {
      "marginVertical": 16,
    }
  }
>
  <Text
    style={
      Object {
        "color": "black",
        "fontFamily": "Signika-Regular",
        "fontSize": 18,
      }
    }
  >
    Date
  </Text>
  <View
    accessible={true}
    collapsable={false}
    focusable={true}
    onBlur={[Function]}
    onClick={[Function]}
    onFocus={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "backgroundColor": "#F4F4F4",
        "borderRadius": 8,
        "color": "#666666",
        "fontFamily": "Signika-Regular",
        "fontSize": 16,
        "marginTop": 8,
        "paddingHorizontal": 24,
        "paddingVertical": 24,
      }
    }
  >
    <Text
      style={Object {}}
    >
      2022-02-04
    </Text>
  </View>
</View>
`;
